site/reference/fn/eval/README.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "`eval`"
+linkTitle: "eval"
+type: docs
+description: >
+  Execute function on resources
+---
+
+<!--mdtogo:Short
+    Execute function on resources
+-->
+
+`eval` executes a function on resources in a directory. Functions are packaged
+as container images or binary executables.
+
+If the function fails (i.e. exits with non-zero status code), `eval` will
+abort and the local filesystem is left intact.
+
+Refer to the [Imperative Function Execution] for detailed overview.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```shell
+kpt fn eval [DIR|-] [flags] [-- fn-args]
```

Comment:
do you explain fn-args anywhere here?

Reply:
Added a brief description. (I need to add it's interplay with fn-config, but need to confirm that before writing it down).